src/app/components/HomePage.tsx

```diff
@@ -52,7 +52,7 @@ const HomePage = () => {
             title={`Find your ${namespaceFindYour[selectedNamespace]}`}
             footer={mission}
           >
-            <section className="uniprot-grid uniprot-grid--centered">
+            <div className="uniprot-grid uniprot-grid--centered">
```

Comment:
`section` should be when the semantics of the area you're using it for make sense to mark it as a section on its own, a landscape of the page, as it might be used by assistive technologies to provide shortcuts to these specific landmarks.
It's a bit of a personal judgment, yes, but here there was clearly too many of them.
As a rule of thumb, I'd try to only use sections when I can define a clear role within one of the supported roles ([see here](https://fae.disability.illinois.edu/rulesets/ROLE_5/)), in which case I specify which role I mean. For example, I did keep one of the sections covered by this PR, but I specified that this area had a `search` role.